### ISSUE TITLE
[i18n] `ng insert-i18n` (reverse `ng extract-i18n`)

### ISSUE THREAD

**Issue by pfeileon** [angular/angular#55073]
### Which @angular/* package(s) are relevant/related to the feature request?

localize

### Description

There is a barrier for using Angular's i18n solution because eventually the source code and source language file will differ and to keep them in sync the source code needs to be manually updated which means a developer has to spend time on an automatable task.

If the process was two-way, the i18n package would be way more useful; and not only for its core intent but also for simply maintaining the text content of an app.

### Proposed solution

Ideally the cli supports a command to write the strings from the extracted source language file back into the source code.

```bash
ng insert-i18n
```

### Alternatives considered

Right now the differences between language file and source code need to be cleaned up manually or by writing a custom automation script. Both alternatives are imho dealbreakers and made me implement a custom way to deal with i18n every single time.

**Comment by pfeileon** [comment]
> Is the below what you are looking for?
> 
> https://github.com/daniel-sc/ng-extract-i18n-merge
> 
> I agree that it should be included in the core i18n library, because i fail to see how the i18n can be useful for real projects without such a tool

No, from my understanding this does not include my requested feature. It's about updating the source language file from the source code but still only one way. If you want this functionality it's a separate feature request.

**Comment by spock123** [comment]
I also ended up writing my own script, because extract only creates new translation files but cannot update existing. So once run 1. time, it's game over basically.

**Comment by martinboue** [comment]
> > Is the below what you are looking for?
> > https://github.com/daniel-sc/ng-extract-i18n-merge
> > I agree that it should be included in the core i18n library, because i fail to see how the i18n can be useful for real projects without such a tool
> 
> No, from my understanding this does not include my requested feature. It's about updating the source language file from the source code but still only one way. If you want this functionality it's a separate feature request.

As this is apparently a different request, I have opened another issue for integrating the "merge" feature natively: https://github.com/angular/angular/issues/56953.
